src/resolve/borders.ts

```diff
@@ -35,8 +35,22 @@ export function border(value: string, theme?: TwTheme): StyleIR | null {
       colorType = `borderRight`;
       break;
   }
+  const hasBorderColor = color(colorType, rest, theme?.borderColor);
+  if (hasBorderColor) {
+    return color(colorType, rest, theme?.borderColor);
+  }
 
-  return color(colorType, rest, theme?.borderColor);
+  // Finally Handle Arbitrary Case
+  // border-[20px] or border-[2.5rem]
+  const prop = `border${direction === `All` ? `` : direction}Width`;
+  rest = rest.replace(/^-/, ``);
+  const numericValue = rest.slice(1, -1);
+  const arbitrary = unconfiggedStyle(prop, numericValue);
```

Comment:
along the same lines, i think naming this `arbitraryWidth` would be a hair more clear.

Reply:
Makes sense. 👍